include/mega/crypto/sodium.h

```diff
@@ -51,7 +92,7 @@ class MEGA_API EdDSA
      * @param data Buffer containing the key bytes.
```

Comment:
the Doxygen doc string does not match the signature (parameters) of the implementation